### PR TITLE
Use STI for indirect assessment types

### PR DESCRIPTION
**Pull request by mrgilman**
* Create separate models, controllers, and forms for each indirect assessment type
* Minimal change to schema -- only requires addition of `type` column to indirect assessments
* Deleted the last of the cucumber specs! :eggplant:

### CODE REVIEW

File: app/models/outcome.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@ class Outcome < ActiveRecord::Base
   belongs_to :course
   has_many :direct_assessments
   has_many :indirect_assessments
```
Comment (derekprior) [inline app/models/outcome.rb]: Can this go away now?

Comment (derekprior) [inline app/models/outcome.rb]: Oh wait, nevermind...

File: config/routes.rb [hunk 2]
```diff
@@ -15,6 +15,9 @@
     resources :assessments
     resources :direct_assessments
     resources :indirect_assessments
```
Comment (derekprior) [inline config/routes.rb]: Can this go away?

**Comment by derekprior** [comment]
Cool. Quick win via STI - this is a a nice improvement.

File: config/routes.rb (hunk 2 above)
Comment (mrgilman) [inline config/routes.rb]: It's in use on the outcomes#show page to display the table of indirect assessments.

Comment (derekprior) [inline config/routes.rb]: :+1:

**Pull request merged**